starport/templates/module/create/oracle/x/{{moduleName}}/oracle.go.plush

```diff
@@ -0,0 +1,52 @@
+package <%= moduleName %>
+
+import (
+    "github.com/bandprotocol/bandchain-packet/obi"
+	"github.com/bandprotocol/bandchain-packet/packet"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	channeltypes "github.com/cosmos/cosmos-sdk/x/ibc/core/04-channel/types"
+	"<%= modulePath %>/x/<%= moduleName %>/types"
+)
+
+func (am AppModule) handleOracleAcknowledgement(ctx sdk.Context, ack channeltypes.Acknowledgement) (sdk.Context, types.RequestID) {
```

Comment:
This is an example of my suggestion above about adding comment.

The difference between `handleOracleAcknowledgement` and `handleOraclePacket` should be described